packages/google_sign_in/android/src/main/java/io/flutter/plugins/googlesignin/GoogleSignInPlugin.java

```diff
@@ -111,7 +148,7 @@ public void onMethodCall(MethodCall call, Result result) {
    * completed (either successfully or in error). This class provides no synchronization consructs
    * to guarantee such behavior; callers are responsible for providing such guarantees.
    */
-  public static final class Delegate {
+  public static final class Delegate implements IDelegate {
```

Comment:
Add `@Override` annotations on the interface methods.

Reply:
Done